libsnark/zk_proof_systems/plonk/plonk.tcc

```diff
@@ -0,0 +1,447 @@
+/** @file
+*****************************************************************************
+Implementation of interfaces for a ppzkSNARK for Plonk.
+
+See plonk.hpp .
+
+*****************************************************************************
+* @author     This file is part of libsnark, developed by SCIPR Lab
+*             and contributors (see AUTHORS).
+* @copyright  MIT license (see LICENSE file)
+*****************************************************************************/
+
+#ifndef PLONK_PPZKSNARK_TCC_
+#define PLONK_PPZKSNARK_TCC_
+
+namespace libsnark
+{
+
+template<typename FieldT> void print_vector(std::vector<FieldT> v)
+{
+    for (size_t i = 0; i < v.size(); ++i) {
+        printf("[%2d]: ", (int)i);
+        v[i].print();
+    }
+}
+
+//
+// Compute the Lagrange basis polynomails for interpolating sets of
```

Comment:
polynomails -> polynomials

Reply:
Fixed.

Reply:
I don't see the fix. Are you sure you pushed it?

Reply:
My apologies, @AntoineRondelet ! I just pushed all my latest changes, including the one in question. Sorry, normally I first apply a fix, mark it as fixed on the PR, accumulate multiple commits like that and push them all together. I now see how this can cause confusion if someone is doing a review at the same time. Anyway, all latest changes are pushed now.